wurst/__legacy__/crafting/MixingPot.jurst

```diff
@@ -47,92 +47,122 @@ function mix takes nothing returns nothing
         endif
+
+        // Twin Islands
         if((yell>2 and blu>2) or (oran>2 and purp>2)) then
             call removeHerbs(u)
-            call UnitAddItemByIdSwapped( ITEM_POTION_TWIN_ISLANDS, u ) // Twin Islands
+            call UnitAddItemByIdSwapped( ITEM_POTION_TWIN_ISLANDS, u )
             return
-	      endif
+          endif
+
+        // Nether
         if(spec and rivStem>1 and butsu>1) then
             call removeHerbs(u)
-            set i = UnitAddItemByIdSwapped( ITEM_NETHER_POTION, u ) // Nether
             set i = UnitAddItemByIdSwapped( ITEM_NETHER_POTION, u )
             return
         endif
+
+        // Drunk
         if(spec and rivRoot>1 and butsu>1) then
             call removeHerbs(u)
-            call UnitAddItemByIdSwapped( ITEM_DRUNKS_POTION, u ) // Drunk
+            call UnitAddItemByIdSwapped( ITEM_DRUNKS_POTION, u )
             return
         endif
+
+        // Fervor
         if(specCount>2 and specTypes>1 and butsu>0) then
             call removeHerbs(u)
-            call UnitAddItemByIdSwapped( ITEM_FERVER_POTION, u ) // Fervor
+            call UnitAddItemByIdSwapped( ITEM_FERVER_POTION, u )
             return
         endif
+
+        // Disease Potion
         if(specTypes>1 and specCount>3 and rivRoot>0) then
             call removeHerbs(u)
-            call UnitAddItemByIdSwapped( ITEM_DISEASE_POTION, u ) //disease pot
+            call UnitAddItemByIdSwapped( ITEM_DISEASE_POTION, u )
             return
         endif
+
+        // Acid Bomb
         if(specTypes>1 and specCount>3 and rivStem>1) then
             call removeHerbs(u)
-            call UnitAddItemByIdSwapped( ITEM_ACID_BOMB, u ) //acid bomb
+            call UnitAddItemByIdSwapped( ITEM_ACID_BOMB, u )
             return
         endif
+
+        // Bee Hive
         if(spec and rivStem>1) then
             call removeHerbs(u)
-            call UnitAddItemByIdSwapped( ITEM_BEE_HIVE, u ) // Bee Hive
+            call UnitAddItemByIdSwapped( ITEM_BEE_HIVE, u )
             return
         endif
+
+        // Dark Rock
         if(rivStem>0 and rivRoot>0 and butsu==2) then
             call removeHerbs(u)
-            call UnitAddItemByIdSwapped( ITEM_DARK_ROCK, u ) // Dark Rock
+            call UnitAddItemByIdSwapped( ITEM_DARK_ROCK, u )
             return
         endif
+
+        // Anabolic Potion
         if(rivStem>3) then
             call removeHerbs(u)
-            call UnitAddItemByIdSwapped( ITEM_ANABOLIC_POTION, u ) // Anabolic
+            call UnitAddItemByIdSwapped( ITEM_ANABOLIC_POTION, u )
             return
         endif
+
+        // Elemental Shield Potion
         if(rivStem>0 and spec and butsu>0) then
             call removeHerbs(u)
-            call UnitAddItemByIdSwapped( ITEM_ELEMENTAL_SHIELD_POTION, u ) // Elemental
+            call UnitAddItemByIdSwapped( ITEM_ELEMENTAL_SHIELD_POTION, u )
             return
         endif
+
+        // Omnicure
 		if(butsu>1 and specTypes>1) then
```

Comment:
The indentation looks weird here

Reply:
Indeed, same with ID.wurst, however it look fine on my side, I don't understand.